www_src/components/el/types/image.jsx

```diff
@@ -2,11 +2,26 @@ var React = require('react/addons');
 var assign = require('react/lib/Object.assign');
 var Spec = require('../../../lib/spec');
 
+var BASE_DEFAULT_URL = 'https://stuff.webmaker.org/webmaker-android/default-images/';
```

Comment:
Also ... curious if this should just be "webmaker" vs. "webmaker-android" as they are shared assets.